matrix/all-in-one.yaml

```diff
@@ -0,0 +1,575 @@
+    comment: Not supported on .NET Framework
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+  - keys:
+      - NPGSQL
+    instrumentedLibraries:
+      - name: Npgsql
+        supportedVersions: 6.0.0 and higher
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+  - keys:
+      - NSERVICEBUS
+    instrumentedLibraries:
+      - name: NServiceBus
+        supportedVersions: 8.0.0 and higher
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+      - metrics:
+          - metric: nservicebus.messaging.successes
+            type: Cumulative counter
+            description: Number of messages successfully processed by the endpoint.
+          - metric: nservicebus.messaging.fetches
+            type: Cumulative counter
+            description: Number of messages retrieved from the queue by the endpoint.
+          - metric: nservicebus.messaging.failures
+            type: Cumulative counter
+            description: Number of messages unsuccessfully processed by the endpoint.
+  - keys:
+      - QUARTZ
+    instrumentedLibraries:
+      - name: Quartz
+        supportedVersions: 3.4.0 and higher
+    comment: Not supported on .NET Framework 4.7.1 and lower
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - name: QuartzNET Instrumentation for OpenTelemetry .NET
+        link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Quartz
+        version: 1.0.0-alpha.3
+        stability: alpha
+    signals:
+      - traces: 
+  - keys:
+      - STACKEXCHANGEREDIS
+    instrumentedLibraries:
+      - name: StackExchange.Redis
+        supportedVersions: 2.0.405 to 3.0.0
+    comment: Not supported on .NET Framework
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - name: StackExchange.Redis Instrumentation for OpenTelemetry
+        link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.StackExchangeRedis
+        version: 1.0.0-rc9.10
+        stability: beta
+    signals:
+      - traces: 
+  - keys:
+      - WCFCLIENT
+      - WCFSERVICE
+    instrumentedLibraries:
+      - name: System.ServiceModel
+        supportedVersions: 4.7.0 and higher of `System.ServiceModel.Primitives`
+    comment: Service side not supported on .NET. `WCFCLIENT for client side instrumentation and `WCFSERVICE` for service side instrumentation
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - name: WCF Instrumentation for OpenTelemetry .NET
+        link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.Wcf
+        version: 1.0.0-rc.12
+        stability: beta
+    signals:
+      - traces: 
+  - keys:
+      - NETRUNTIME
+    instrumentedLibraries:
+      - name: .NET runtime
```

Comment:
Technically it is not a library, but the process/runtime, similar for the `PROCESS`.

@pellared, any idea for better naming instead of "instrumented libraries"?

Reply:
Ha, hard to find a word for all. We use "frameworks and libraries" in the docs, but that's too long for a YAML key. We could also just use `component`, similar to OTel Collector nomenclature.

Reply:
I will change to instrumentedComonents

Reply:
8d8f886148cfd18d2f8d3769099a3755900f8ef8